apps/condo/domains/subscription/schema/SubscriptionPlan.js

```diff
@@ -0,0 +1,135 @@
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { ORGANIZATION_TYPES } = require('@condo/domains/organization/constants/common')
+const access = require('@condo/domains/subscription/access/SubscriptionPlan')
+
+const ERRORS = {
+    TRIAL_DAYS_MUST_BE_NON_NEGATIVE: {
+        code: BAD_USER_INPUT,
+        type: 'TRIAL_DAYS_MUST_BE_NON_NEGATIVE',
+        message: 'trialDays must be >= 0',
+    },
+}
+
+const SubscriptionPlan = new GQLListSchema('SubscriptionPlan', {
+    schemaDoc: 'Subscription plan that defines a product with features. Prices are defined via SubscriptionPlanPricingRule',
+    fields: {
+
+        name: {
+            schemaDoc: 'Human-readable name of the subscription plan',
+            type: 'Text',
+            isRequired: true,
+        },
+
+        description: {
+            schemaDoc: 'Detailed description of the subscription plan',
+            type: 'Text',
+            isRequired: false,
+        },
+
+        organizationType: {
+            schemaDoc: 'Type of organization this plan is available for',
+            type: 'Select',
+            options: ORGANIZATION_TYPES,
+            dataType: 'string',
+            isRequired: true,
+            access: {
+                read: true,
+                create: true,
+                update: false,
+            },
+        },
+
+        isHidden: {
+            schemaDoc: 'Whether this plan is hidden and not available for purchase',
+            type: 'Checkbox',
+            defaultValue: false,
+            isRequired: true,
+        },
+
+        trialDays: {
+            schemaDoc: 'Number of trial days for this plan. If 0, trial activation is not available for this plan',
+            type: 'Integer',
+            defaultValue: 0,
+            isRequired: true,
+            hooks: {
+                validateInput: async ({ resolvedData, fieldPath, existingItem, context }) => {
+                    const trialDays = resolvedData[fieldPath] !== undefined ? resolvedData[fieldPath] : existingItem?.trialDays
+
+                    if (trialDays !== null && trialDays !== undefined && trialDays < 0) {
+                        throw new GQLError(ERRORS.TRIAL_DAYS_MUST_BE_NON_NEGATIVE, context)
+                    }
+                },
+            },
+        },
+
+        priority: {
+            schemaDoc: 'Priority for sorting plans on frontend and selecting active subscription (higher values = higher priority)',
+            type: 'Integer',
+            defaultValue: 0,
+            isRequired: true,
+        },
+
+        news: {
+            schemaDoc: 'Whether news feature is included in this plan',
+            type: 'Checkbox',
+            defaultValue: false,
+            isRequired: true,
+        },
+
+        marketplace: {
+            schemaDoc: 'Whether marketplace feature is included in this plan',
+            type: 'Checkbox',
```

Comment:
What about quantitative restrictions?
For example, on the number of employees in the organization or something else

Reply:
We can add Int field for this type of restriction or mb we do dynamic restrictions (task 12735)